.github/ISSUE_TEMPLATE/new_proposal.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Idea proposal
+about: Suggest an idea for this project
+title: "[TITLE]"
+labels: proposal
+assignees: ''
+
+---
+
+# [TITLE]
+
+## Metadata
+
+This proposal is (tick applicable):
+
+* [ ] A new feature
+* [ ] an extension to an existing RFC/proposal <LINK>
+* [ ] an alternative approach to an existing RFC or proposal <LINK>
+
+### Change log
```

Comment:
The contents below does not look like a change log. Maybe a different title here?